app/controllers/advocates/claims_controller.rb

```diff
@@ -8,8 +9,8 @@ def landing; end
 
   def index
     claims = @context.claims.order(created_at: :desc)
-    claims = claims.find_by_advocate_name(params[:search_advocate]) if params[:search_advocate].present?
-    claims = claims.find_by_defendant_name(params[:search_defendant]) if params[:search_defendant].present?
+    claims = claims.find_by_advocate_name(params[:search]) if ['All', 'Advocate'].include?(params[:search_field])
```

Comment:
see my main comment - this code (i believe) makes an `All` search (signed in as an advocate) filter by advocate and defendant name despite not having an 'Advocate' search field option